### ISSUE TITLE
method to create training sets with mock data

### ISSUE THREAD

**Issue by gschwend** [linea-it/pzserver#102]
Implement a method in pzserver lib to create training sets with mock data from the parquet files stored on Lustre.

```[tasklist]
### Tasks
- [x] default method = random selection
- [ ] https://github.com/linea-it/pzserver/issues/103
```

**Comment by deborajanini** [comment]
@gschwend qual o tamanho desta issue?

**Comment by gschwend** [comment]
@deborajanini large (já atualizei), também coloquei a segunda parte em um issue separado (em backlog)

**Comment by luigilcsilva** [comment]
Movendo para Blocked. Razão: falta de tracts na tabela matches_truth. O método que eu queria usar para cumprir essa issue era:
1) Fazer o merge da tabela object com a tabela truth, usando a matches_object como intermediária (script já está pronto)
2) Selecionar uma sample aleatória de uns 4,000,000 de objetos.
3) Gerar um training set degradado com GridSelection do rail.
4) Gerar um training set randômico de mesmo tamanho do training set degradado a partir desses 4,000,000 de objetos.
5) Rodar o pipeline do pz_compute para os training sets dos passos 4 e 5, e comparar as métricas.

**Comment by luigilcsilva** [comment]
arquivos em `/lustre/t1/cl/lsst/dp02/primary/catalogs` 

object 
157 arquivos (tracts)  [2897, … , 5074] 
coadd_patches/dp02_CoaddPatches.csv
157 unique tracts [2897, … , 5074] 
truth_match
166 tracts (dentro de truth_summay, junto com truth_galaxy) [2723, … , 5074]
9 tracts de diferença 
sobrando 14 tracts:  [2723, … , 2735, 2896]
faltando 5 tracts [4848, 4849, 5062, 5063, 5064]
matches_truth
152 arquivos (tracts)  - comparando com a tabela objects, os tracts faltantes são: [4848, 4849, 5062, 5063, 5064]

Estamos cientes dos tracts faltantes. Seguiremos assim.

**Comment by luigilcsilva** [comment]
Passos:
1) Gerar um test set a partir de um sample do catálogo inteiro.
2) Gerar um sample que não inclua valores do test set anterior.
3) Em cima dessa sample, fazer um training set degradado e outro aleatório.
4) Rodar o pz_compute e comparar as métricas.
